released/packages/coq-library-undecidability/coq-library-undecidability.1.0.1+8.16/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+version: "1.0.1+8.16"
+maintainer: "forster@cs.uni-saarland.de"
+homepage: "https://github.com/uds-psl/coq-library-undecidability/"
+dev-repo: "git+https://github.com/uds-psl/coq-library-undecidability/"
+bug-reports: "https://github.com/uds-psl/coq-library-undecidability/issues"
+authors: ["Yannick Forster"
+          "Dominique Larchey-Wendling"
+          "Andrej Dudenhefner"
+          "Edith Heiter"
+          "Marc Hermes"
+          "Johannes Hostert"
+          "Dominik Kirst"
+          "Mark Koch"
+          "Fabian Kunze"
+          "Gert Smolka"
+          "Simon Spies"
+          "Dominik Wehr"
+          "Maximilian Wuttke"]
+
+license: "CeCILL-2.1"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" {= "8.16.0"}
```

Comment:
Is there an obvious reason now to allow the (potential) 8.16.1 release? The Coq Core Team generally guarantees compatibility.
```suggestion
  "coq" {>= "8.16.0" & < "8.17"}
```

Reply:
No reason :)